src/mutations/ScheduledEventMutations.ts

```diff
@@ -177,13 +176,17 @@ export default class ScheduledEventMutations {
   @Authorized([Roles.USER_OFFICER, Roles.INSTRUMENT_SCIENTIST])
   async activate(
     ctx: ResolverContext,
-    { id }: ActivateScheduledEventInput
+    { id }: { id: number }
```

Comment:
You could do Pick<ActivateScheduledEventInput', 'id'>

Reply:
@jekabs-karklins the thing is that now that type it is called `ActivateScheduledEventsInput` and it contains `ids` as `number[]` array and here I use only one id.